### DISCUSSION TITLE
AG620 digital works

### DISCUSSION THREAD

**Discussion by ravenblackdusk** [Nortank12/deepcool-digital-linux#6] [General]
all modes work, I've also enabled alarm, but i'm not comfortable bringing cpu temp to 85(and don't know how). I don't have a reddit user so I wrote this here, hope you see it @Nortank12 @asdfzdfj

**Comment by Nortank12** [comment]
I see it, thank you. Don't worry about the temperature. If everything else matches, it should work as well.

**Reply by ravenblackdusk** [reply]
yes, it seems the alarm also works. the temperature blinks after 85
